recipe/meta.yaml

```diff
@@ -8,21 +8,24 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.{{ bundle }}
-  sha256: af372d03c53c763460e963fb129afe2b123ff21d84e81c27d16460c77d843201
+  # url: https://pypi.org/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.{{ bundle }}
+  url: https://github.com/xnd-project/rbc/archive/refs/tags/v{{ version }}.{{ bundle }}
+  sha256: 0a58aac7464598cef6838e7949ebe43fc85a8901f241d423239b24c477e45fd5
```

Comment:
@pearu, I manage to build RBC using the GitHub tarball. But from previous experience using it as tarball source, the SHA256 is unreliable and require manual intervention when the regro bot updates the feedstock. What do you think of using github for just this release and we can fix the tarball issue in RBC to use PyPI in the next one?

Reply:
@guilhermeleobas If the release tar-ball of rbc 0.7.0 does not contain _rbclib.h file then the release is broken anyway. So I suggest fixing the 0.7.0 tar-ball or creating a bug-fix release 0.7.1.